### ISSUE TITLE
Mann vs. Machine Non-stop Queuing and another bug.

### ISSUE THREAD

**Issue by SampleTek** [mastercomfig/tf2-patches#557]
I'm not sure if anyone's posted anything about this, but frequently whenever I try queuing for an MVM Boot Camp match, **it immediately finds another match for me to join** as soon as I load in to the server so I have to cancel the queue. At the same time, doing this _kicks me out of the server I'm already in._ I've tried riding it out only for it to queue more and more matches that I'm forced to join. It's extremely frustrating when all I wanna do is play the game!

Also, I've discovered the hard way that if you queue for Boot Camp and close the 'find a game' menu, it will **only display the list in a way so that only the missions' difficulties are shown.** This is also the case if I select Mann Up mode, but for all I know about programming it could just be a graphical error.

These are issues in the vanilla game and I hope to seen them fixed if Valve decides to implement your changes. If these bugs were fixed already, I'm sorry for not paying attention to patch notes. Y'all are doing God's work, and I have nothing against what you're doing here for our beloved game!

**Comment by treacherousfiend** [comment]
Unfortunately we cannot fix matchmaking issues like this, since the build that we are working on is from before JI's release, which added the new matchmaking.

**Comment by SampleTek** [comment]
I understand. I'll post a second issue separately from this, and hopefully I can figure out how to use a tag! Thank you, and happy holidays.

**Comment by SampleTek** [comment]
UPDATE: False Alarm! (maybe)

While I was trying to take a screenshot for proof that the glitch was real, my screen has been incessantly flickering for the past year and so it messed up any attempts at a screenshot. I decided to finally try and find a way to fix it online, even though I believed it was a lost cause. After I tried enabling vertical sync, not only did the flickering stop, but it also made me realize that perhaps the glitch was caused by the flickering itself. My method for getting the 'glitch' to happen never worked, but if it does, I'll be sure to make another post, this time with the correct tags (hopefully) and proper steps as to how it was discovered.

I'll keep this issue open for any further comments on the matter.
